cmd/docker/docker.go

```diff
@@ -17,9 +19,14 @@ import (
 	"github.com/docker/docker/pkg/term"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
-	"strings"
 )
 
+type hideOptions struct {
```

Comment:
We use this `Option` naming convention for data objects populated from command line flags. This struct is not being used that way, so I think the name is confusing.

How about `apiProperties` or `apiFeatures` or something like that?